ff/serving/provider/postgres.go

```diff
@@ -142,6 +151,14 @@ func (q postgresSQLQueries) createValuePlaceholderString(columns []TableColumn)
 }
 
 func (q postgresSQLQueries) trainingSetCreate(store *sqlOfflineStore, def TrainingSetDef, tableName string, labelName string) error {
+	return q.trainingSetGeneral(store, def, tableName, labelName, false)
+}
+
+func (q postgresSQLQueries) trainingSetUpdate(store *sqlOfflineStore, def TrainingSetDef, tableName string, labelName string) error {
+	return q.trainingSetGeneral(store, def, tableName, labelName, true)
+}
+
+func (q postgresSQLQueries) trainingSetGeneral(store *sqlOfflineStore, def TrainingSetDef, tableName string, labelName string, isUpdate bool) error {
```

Comment:
Can we get a better name for this, trainingSetQuery?

Reply:
Changed